tests/cue/env.py

```diff
@@ -6,36 +6,19 @@
 class VSCEnvTest(rfm.RunOnlyRegressionTest):
     descr = "test environment variable "
     envar = parameter(['VSC_HOME', 'VSC_DATA', 'VSC_SCRATCH'])
-    valid_systems = required
-    valid_prog_environs = required
+    valid_systems = ["*"]
```

Comment:
I am not sure `["*"]` is the best choice here; the intention was to run this test on a login node, not inside a job. With the current settings file, `["*:local"]` seems a better choice to me.

Reply:
isn't it useful to also check this in a job, to make sure the envars are available there too?

Reply:
Agree with sam, will add *:single-node in the list